server/app.ts

```diff
@@ -51,6 +52,12 @@ export default function createApp(
   // Server Configuration
   app.set('port', config.port)
 
+  // Reads config from SENTRY_DSN env variable, if exists
+  Sentry.init({ environment: config.sentry.environment })
```

Comment:
I tried to add this in `server/config.ts` as `dsn: get('SENTRY_DSN', 'dummy_dsn', requiredInProduction)` but ran into problems:

1. if the DSN is invalid, Sentry will actively refuse it, leading to boot errors
2. I cannot have `null` values with the `get` function and it did not feel like adding optionality into it is a good idea

So I'm going with the [documented effect](https://docs.sentry.io/platforms/node/guides/express/configuration/options/#dsn):
> The _DSN_ tells the SDK where to send the events. If this value is not provided, the SDK will try to read it from the `SENTRY_DSN` environment variable. If that variable also does not exist, the SDK will just not send any events.